pkg/apiserver/api/v1/event.proto

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/protobuf/any.proto";
+
+message Event {
+  google.protobuf.Any object = 1;
+  string type = 2;
```

Comment:
https://developers.google.com/protocol-buffers/docs/proto3#enum

use enum for Event type (and as I've said in later comment, this should be typed, not untyped)

Reply:
replied here: https://github.com/kubermatic/kubecarrier/pull/437#discussion_r432352448

Reply:
here enum is the right choice since it's unlikely members would change in the future, and a client should make a specific action depending on the event type.